examples/chip-tool/commands/pairing/Commands.h

```diff
@@ -165,6 +165,14 @@ class PairBleThread : public PairingCommand
     {}
 };
 
+class PairBleWiFiOrThread : public PairingCommand
```

Comment:
The "ble" commands are basically deprecated.  People should be using the "code" commands.  So this should really add "code-wifi-thread" if we add something here.  If we strongly feel that ble-wifi-thread is also useful, we can add that too, but the "code" version is required.

Reply:
Added `code-wifi-thread` command.